mappings/net/minecraft/world/gen/feature/PlacedFeature.mapping

```diff
@@ -1,18 +1,42 @@
 CLASS net/minecraft/unmapped/C_jewbaoib net/minecraft/world/gen/feature/PlacedFeature
 	FIELD f_czdlpzzg CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_oyewqkwq LIST_OF_LISTS_CODEC Lcom/mojang/serialization/Codec;
 	FIELD f_qarnvrms feature Lnet/minecraft/unmapped/C_cjzoxshv;
 	FIELD f_qdvzmuxj placementModifiers Ljava/util/List;
 	FIELD f_rzzjrmle REGISTRY_CODEC Lcom/mojang/serialization/Codec;
 	FIELD f_zikplwuy LIST_CODEC Lcom/mojang/serialization/Codec;
+	METHOD <init> (Lnet/minecraft/unmapped/C_cjzoxshv;Ljava/util/List;)V
+		ARG 1 feature
+		ARG 2 placementModifiers
+	METHOD equals (Ljava/lang/Object;)Z
+		ARG 1 o
 	METHOD m_bausfbxg getDecoratedFeatures ()Ljava/util/stream/Stream;
+	METHOD m_cdvmhesi feature ()Lnet/minecraft/unmapped/C_cjzoxshv;
 	METHOD m_ihlizohn (Lnet/minecraft/unmapped/C_jewbaoib;)Lnet/minecraft/unmapped/C_cjzoxshv;
 		ARG 0 feature
+	METHOD m_jqhgzpkw placementModifiers ()Ljava/util/List;
 	METHOD m_jtpyejkp (Lnet/minecraft/unmapped/C_jewbaoib;)Ljava/util/List;
 		ARG 0 feature
 	METHOD m_myuhmsgj place (Lnet/minecraft/unmapped/C_ldkphtbr;Lnet/minecraft/unmapped/C_kgfalbim;Ljava/util/Random;Lnet/minecraft/unmapped/C_hynzadkk;)Z
 		ARG 1 world
 		ARG 2 chunkGenerator
 		ARG 3 random
 		ARG 4 pos
+	METHOD m_qncvixll placeWithContext (Lnet/minecraft/unmapped/C_ldkphtbr;Lnet/minecraft/unmapped/C_kgfalbim;Ljava/util/Random;Lnet/minecraft/unmapped/C_hynzadkk;)Z
+		ARG 1 world
+		ARG 2 chunkGenerator
+		ARG 3 random
+		ARG 4 pos
+	METHOD m_xpycbbci placeWithContext (Lnet/minecraft/unmapped/C_hszwdqtd;Ljava/util/Random;Lnet/minecraft/unmapped/C_hynzadkk;)Z
+		ARG 1 decoratorContext
+		ARG 2 random
+		ARG 3 pos
 	METHOD m_xzvkggoc (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/kinds/App;
 		ARG 0 instance
+	CLASS C_nehzjnhe Test
```

Comment:
Should probably note that this class is literally named `test` in mojmap, and isn't used in production anywhere

Reply:
I swear this is mapped in another PR, but I can't figure out which